.bazelci/presubmit.yml

```diff
@@ -156,6 +162,34 @@ tasks:
     # TODO(gregmagolan): make node_repositories acccept different archives for different platforms
     - "//examples:examples_vendored_node"
     - "//examples:examples_vendored_node_and_yarn"
+  ubuntu1804_cross_compile_darwin:
+    name: ubuntu1804_cross_compile_darwin
+    platform: ubuntu1804
+    # Build on linux with the node --platform set to darwin
+    build_flags:
+    - "--platforms=@build_bazel_rules_nodejs//toolchains/node:darwin_amd64"
+    build_targets:
+    - "//..."
```

Comment:
feels really expensive to re-build everything in each configuration. I wonder if we are using more CI resources than any other rules

Reply:
we do have quite a few jobs now. it's probably not strictly necessary to build in every permutation but I think at the least we should build everything or some subset of everything with each platform.

Reply:
maybe just `//internal/...` is sufficient to get the coverage we need?

Reply:
👍 that should do it for now as only node_binary/node_test vary the build based on platform. We can expand coverage in the future if some of the packages with build differences based on platform.